includes/export.form.inc

```diff
@@ -245,7 +245,7 @@ function islandora_datastreams_io_export_form_submit(array $form, array &$form_s
     }
     else {
       $bad_pids = $good_pids = array();
-      $pids_arr = explode("\n", $pids);
+      $pids_arr = explode(PHP_EOL, $pids);
       foreach ($pids_arr as $pid) {
         $pid = str_replace("\\r", "", trim($pid));
```

Comment:
This raises questions.  Is this really a replace of literal '\r' , or is this a replace of CRs unhandled by the LF explode?

Reply:
AFAIK the strings are harvested internally.  We could explode on a regex if we didn't know where the source files were coming from.

Reply:
The input of PID values would be whatever the user entered in the textarea and I am not an expert on all of the operating systems' copy/paste formats -- but did have to put this in there to clean up the $pid value per line because one editor did not have the \r at all and another one did. I would think that the changes would certainly not break anything - possibly the str_replace on line 250 is harmlessly redundant.